requirements_test.txt

```diff
@@ -2,7 +2,7 @@
 -r requirements_test_min.txt
 coveralls~=3.0
 coverage~=5.5
-pre-commit~=2.12
+pre-commit~=2.12;python_full_version>="3.6.2"
```

Comment:
Why not use just `python_version`? Same in `requirements_test_pre_commit.txt`.
https://www.python.org/dev/peps/pep-0496/#version-numbers

Reply:
`platform.python_version()` is a dotted string, so according to that PEP `python_version` doesn’t include the micro part.

Reply:
Makes sense, didn't know that